lib/process-services-cloud/src/lib/process/start-process/components/start-process-cloud.component.html

```diff
@@ -100,14 +100,15 @@
 <ng-template #taskFormCloudButtons>
     <div class="adf-start-process-cloud-actions">
         <button
-            *ngIf="showCancelButton"
+            *ngIf="showCancelButton && displayCancelButton"
```

Comment:
show cancel and display cancel are literally the same words

Reply:
Removed displayCancel button